generatedocs/data/params.json

```diff
@@ -456,6 +459,12 @@
     "randomaction.actions.overrides": [
         "(optional) Static overrides to the action. The overrides can include any or all of the settings from the original action, as determined by the `type` field. If nothing is specified, the default values are used."
     ],
+    "reconnectsettings.reconnect" : [
+        "Enable reconnect attempt when websocket gets disconnected, defaults to false."
```

Comment:
Suggestion:
"Enable re-connection attempts if the WebSocket is disconnected. Defaults to false."